commit e38b3dabdab22246d6a3963b567e064c542ce9d9
Author: Mitchell Augustin <mitchell.augustin@canonical.com>
Date:   Tue Jul 22 08:34:44 2025 -0500

    wpa_supplicant: Bump DEFAULT_BSS_MAX_COUNT to 1000
    
    Many congested areas have more than 200 APs in range of a typical
    device. When the number of APs exceeds BSS_MAX_COUNT, even nearby APs
    can be absent from the scanning device's AP list. Bump the default to
    1000 to be more in line with a realistic modern maximum number of APs in
    a given area.
    
    Discussion: https://lists.infradead.org/pipermail/hostap/2025-July/043612.html
    
    Signed-off-by: Mitchell Augustin <mitchell.augustin@canonical.com>

diff --git a/wpa_supplicant/config.h b/wpa_supplicant/config.h
index c187d36ed..4e3997877 100644
--- a/wpa_supplicant/config.h
+++ b/wpa_supplicant/config.h
@@ -29,7 +29,7 @@
 #define DEFAULT_P2P_INTRA_BSS 1
 #define DEFAULT_P2P_GO_MAX_INACTIVITY (5 * 60)
 #define DEFAULT_P2P_OPTIMIZE_LISTEN_CHAN 0
-#define DEFAULT_BSS_MAX_COUNT 200
+#define DEFAULT_BSS_MAX_COUNT 1000
 #define DEFAULT_BSS_EXPIRATION_AGE 180
 #define DEFAULT_BSS_EXPIRATION_SCAN_COUNT 2
 #define DEFAULT_MAX_NUM_STA 128
